src/app/features/checkout/components/payment/payment.component.ts

```diff
@@ -64,4 +84,12 @@ export class PaymentComponent implements OnInit {
     }
     return '';
   }
+
+  checkout() {
+    console.log(this.paymentForm.value);
```

Comment:
Can't we open the lease request here?
We have all the queries, we only need to send a request, am I wrong?

Reply:
Yes this is the next thing I am going to do